package.json

```diff
@@ -26,8 +26,7 @@
     "yeoman-generator",
     "generator",
     "boilerplate",
-    "danger",
-    "danger-plugin"
+    "danger"
```

Comment:
This shouldn't really show up in results for danger-plugin. https://www.npmjs.com/search?q=keywords:danger-plugin&page=1&ranking=optimal